.github/workflows/ci.yml

```
@@ -221,7 +226,7 @@ jobs:
           name: ipfs-webui_${{ github.sha }}.car
 
       - name: Dry-run semantic release
-        if: github.ref != 'refs/heads/main'
+        if: github.ref != 'refs/heads/main' || github.event.inputs.kubo-version
```

Should we be skipping this step as well if `kubo-version` is provided? It failed during my test run: https://github.com/ipfs/ipfs-webui/actions/runs/3135588411

```suggestion
        if: github.ref != 'refs/heads/main' && !github.event.inputs.kubo-version
```

---

The error was 

```
SemanticReleaseError: The release branches are invalid in the `branches` configuration.
```
I don't think we should skip the dry-run of semantic release as this helps determines whether the PR is going to break the real semantic release. 

We could update https://github.com/ipfs/ipfs-webui/blob/d3a652400902e564ebae5f257acfd254ee52472a/.github/workflows/ci.yml#L228 if we need to. It seems like maybe `github.ref` was empty?

---

we could probably just change the `if: ` to check if it's a workflow_dispatch or not

---

considering this change is specifically for running workflow_dispatch with a requested kubo version, I'm fine with the `&& !` change.